src/LibraryView.tsx

```diff
@@ -61,6 +63,17 @@ export class LibraryView {
                 thisObject.layoutSpecsJson = JSON.parse(jsonString);
                 thisObject.updateContentsInternal();
             });
+
+        // fetch search items (in json format) if queryString exists and is not empty
+        if (queryString && queryString.length > 0) {
+            fetch(queryString)
+                .then(function (response: Response) {
+                    return response.text();
+                }).then(function (jsonString) {
+                    thisObject.searchItemsJson = JSON.parse(jsonString);
+                    thisObject.updateContentsInternal();
+                });
+        }
```

Comment:
Hey @Benglin we don't have http endpoint for REACH.  You should use  Command /Message pattern for the requests.